### PR TITLE
Created a new repo and branch for review check

### PR DESCRIPTION
**Pull request by ikoote1**
#To do list project.
- Created a new repo and branch for review check

#Features
- Add an activity to the to-do list.
- You can delete an activity from the to-do list.
- You can edit the activity from the to-do list.

Thanks for reviewing my work

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    <title>To Do list</title>
+  </head>
+  <body>
+    <main id="main">
+      <div>
+        <div class="todo-title flex">
+          <h1>Today's To Do</h1>
+          <i class="sync-btn fa-solid fa-rotate"></i>
+        </div>
+        <hr />
```
Comment (lorenzoworx) [inline src/index.html]: Hey Ikoote, I would suggest you use the `<hr>` tag for the horizontal rule the forward slash was necessary for previous versions of HTML but it was removed in HTML 5

**Review by lorenzoworx — COMMENTED** [review]
Great job Ikoote, your code is amazing 👌.  I found a little issue within your HTML, nothing major though

**Comment by ikoote1** [comment]
Thanks for reviewing my work.

**Review by lorenzoworx — APPROVED** [review]
Well done Ikoote 👏

**Pull request merged**